src/Components/Placeholders/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "../ProgressBar";
```

Comment:
Why does it exports something not it's folder?

The idea of index, is that it is index of this exact module. It can not export something else.